Today Extension/TodayExtensionCell.swift

```diff
@@ -0,0 +1,220 @@
+        destinationLabel.snp.makeConstraints { make in
+            let offset: CGFloat = 2.0
+            let bottom: CGFloat = 50.0
+
+            make.top.equalTo(departureLabel.snp.bottom).offset(offset)
+            make.bottom.equalToSuperview().inset(bottom)
+            make.leading.equalTo(departureLabel)
+            make.trailing.lessThanOrEqualToSuperview().inset(rightMargin)
+        }
+
+        contentView.addSubview(liveBackground)
+        contentView.addSubview(liveLabel)
+        contentView.addSubview(liveIndicatorView)
+        if showLiveElements {
+            liveLabel.snp.makeConstraints { make in
+                let vertical: CGFloat = 9.0
+                let inset: CGFloat = 8.0
+
+                make.top.equalTo(destinationLabel.snp.bottom).offset(vertical)
+                make.bottom.equalToSuperview().inset(verticalMargin)
+                make.leading.equalTo(departureLabel).inset(inset)
+                make.trailing.lessThanOrEqualTo(rightMargin)
+            }
+
+            liveIndicatorView.snp.makeConstraints { make in
+                let offset: CGFloat = 8.0
+
+                make.centerY.equalTo(liveLabel.snp.centerY)
+                make.leading.equalTo(liveLabel.snp.trailing).offset(offset)
+                make.trailing.lessThanOrEqualTo(rightMargin)
+                make.height.equalTo(liveIndicatorView.intrinsicContentSize.height)
+            }
+
+            liveBackground.snp.makeConstraints { make in
+                let offset: CGFloat = 8.0
+                let border: CGFloat = 2.0
+
+                make.leading.equalTo(departureLabel)
+                make.top.equalTo(liveLabel).offset(-border)
+                make.bottom.equalTo(liveLabel).offset(border)
+                make.trailing.equalTo(liveIndicatorView).offset(offset)
+            }
+        }
+
+    }
+
+    func configure(route: Route?, destination: String, top: CGFloat, bottom: CGFloat) {
+        destinationName = destination
+        if let route = route {
+            self.route = route
+            if let departDirection = (route.directions.filter { $0.type == .depart }).first {
+                busDirection = departDirection
+                busIcon = BusIcon(type: .directionSmall, number: departDirection.routeNumber)
+                contentView.addSubview(busIcon!)
+
+                if let busIcon = busIcon {
+                    let bottom: CGFloat = 66.0
+                    let offset: CGFloat = 18.0
+
+                    busIcon.snp.makeConstraints { make in
+                        make.top.equalToSuperview().inset(verticalMargin)
+                        make.bottom.equalToSuperview().inset(bottom)
+                        make.leading.equalToSuperview().inset(offset)
+                        make.width.equalTo(busIcon.intrinsicContentSize.width)
+                    }
+                }
+
+                setUpDepartureLabel()
+                setUpDestinationLabel()
+                setUpLiveElements()
+
```

Comment:
could we add a `return` here and then just do `setupNoRoute()` once instead of twice